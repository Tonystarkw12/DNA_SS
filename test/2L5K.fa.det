Linear DNA folding at 37° C. [Na+] = 1.0, [Mg++] = 0.0.
 
Structure    1
 
2L5K
 dG =     -1.57  dH =    -23.00  dS =    -69.10  Tm =   59.7 â„ƒ
 
 
External loop:	ddG =  -1.05  12 ss bases &  1 closing helices.
Stack:		ddG =  -0.88 External closing pair is A(     7)-T(    17)
Stack:		ddG =  -1.30 External closing pair is T(     8)-A(    16)
Stack:		ddG =  -1.84 External closing pair is C(     9)-G(    15)
Helix:		ddG =  -4.02   4 base pairs.
Hairpin loop:	ddG =   3.50          Closing pair is C(    10)-G(    14)
