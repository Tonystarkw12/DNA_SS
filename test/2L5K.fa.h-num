   level  length istart jstart h-num                                            
      1      4      7     17    1.0
