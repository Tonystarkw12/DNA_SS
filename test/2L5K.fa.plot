   level  length istart jstart energy
      1      4      7     17    -15
