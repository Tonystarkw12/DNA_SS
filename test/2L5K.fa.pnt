2L5K                                                       
#BASES=       23
         10         20         30
 CAGTTGATCC TTTGGATACC CTG
