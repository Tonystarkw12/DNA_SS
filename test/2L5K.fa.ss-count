           1
           1           1  C
           2           1  A
           3           1  G
           4           1  T
           5           1  T
           6           1  G
           7           0  A
           8           0  T
           9           0  C
          10           0  C
          11           1  T
          12           1  T
          13           1  T
          14           0  G
          15           0  G
          16           0  A
          17           0  T
          18           1  A
          19           1  C
          20           1  C
          21           1  C
          22           1  T
          23           1  G
