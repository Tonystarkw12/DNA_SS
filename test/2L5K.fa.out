Linear DNA folding at 37° C. [Na+] = 1.0, [Mg++] = 0.0.
 
Structure    1
 
Folding bases      1 to     23 of 2L5K
 dG =     -1.57  dH =    -23.00  dS =    -69.10  Tm =   59.7 â„ƒ
 
 
        10
CAGTTG|   T
      ATCC \
      TAGG T
GTCCCA^   T
  20
 
 
